openedx/stanford/lms/urls.py

```diff
@@ -0,0 +1,41 @@
+from django.conf import settings
+from django.conf.urls import url
+
+
+urlpatterns = (
+    url(
+        r'^course_sneakpeek/{}/$'.format(
+            settings.COURSE_ID_PATTERN,
+        ),
+        'student.views.setup_sneakpeek',
+        name='course_sneakpeek',
+    ),
+    url(
+        r'^get_analytics_answer_dist/',
+        'courseware.views.views.get_analytics_answer_dist',
+        name='get_analytics_answer_dist',
+    ),
+)
+if settings.SHIB_ONLY_SITE:
+    urlpatterns += (
```

Comment:
These no longer function due to the FORCED_TPA_PROVIDER_ID effectively preventing the default logistration page from ever showing, perhaps we can just remove these if their use case is obsolete or can be achieved elsewise

Reply:
Should this wait on #733